_topic_map.yml

```diff
@@ -279,18 +279,8 @@ Topics:
     File: installing-rhv-user-infra
   - Name: Uninstalling a cluster on RHV
     File: uninstalling-cluster-rhv
-- Name: Installing on oVirt
-  Dir: installing_rhv
-  Distros: openshift-origin
-  Topics:
-  - Name: Installing a cluster quickly on oVirt
-    File: installing-rhv-default
-  - Name: Installing a cluster on oVirt with customizations
-    File: installing-rhv-customizations
-  - Name: Installing a cluster on oVirt with user-provisioned infrastructure
-    File: installing-rhv-user-infra
-  - Name: Uninstalling a cluster on oVirt
-    File: uninstalling-cluster-rhv
+  - Name: Installing a cluster on RHV in a restricted network
+    File: installing-rhv-restricted-network
```

Comment:
I would recommend moving this above the `Uninstalling a cluster on RHV` assembly, to follow consistent ordering of other installer sections.